CONTRIBUTING.md

```diff
@@ -2,28 +2,27 @@
+If you're interested in contributing a project to CNCF, please see the full project proposal process is located [here](https://github.com/cncf/toc/blob/main/process/project_proposals.md).
 
-The full project proposal process is located [here](https://github.com/cncf/toc/blob/main/process/project_proposals.md).
+## Contributing to Technical Advisory Groups and Working Groups
 
-## TOC Contributors
-
-There has been a call from CNCF’s Technical Oversight Committee (TOC) for additional contributors and expertise to help evaluate potential projects and contribute to [CNCF TAGs](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) and [working groups](https://github.com/cncf/toc#working-groups). With the metaphor of the TOC as an open source project and the TOC [members](https://github.com/cncf/toc#members) as the maintainers, we are making a call for new **TOC Contributors**.
+There has been a call from CNCF’s Technical Oversight Committee (TOC) for contribute to [CNCF TAGs](https://github.com/cncf/toc/blob/main/tags/cncf-tags.md) and [working groups](https://github.com/cncf/toc#working-groups).
 
 Possible ways to contribute:
 
+* Technical content and leadership
 * Tech due diligence for projects
+* White papers
 * Time spent helping projects
-* Liaison with GB
-* CNCF TAGs & working groups (various tasks)
-* Technical content for website
 
-If you are interested in engaging in this way, we would encourage you to issue a pull request to [TOC Contributors](https://github.com/cncf/toc/blob/main/CONTRIBUTORS.md) that you desire to become a TOC Contributor. Although there is not an actual limit of having one Contributor per company, we would encourage CNCF member companies to designate an official TOC Contributor who is tasked with consulting internal experts and expressing a semi-official view on a given project. We will list current TOC Contributors on a page similar to https://www.cncf.io/people/ambassadors/.
+## Getting involved in the TOC itself
+
+We use the [cncf/toc](https://github.com/cncf/toc/) github repository to track ongoing work and discussions. please see [README.md](https://github.com/cncf/toc#readme) for all the communication channels. We welcome the community to join us in our public meetings as well.
 
 This is not only about individual contribution.  It is also about rallying help from your employer, e.g., if you work for a CNCF Member company. Given the [breadth](https://landscape.cncf.io/) of projects represented by cloud native, it is impossible for anyone to be an expert in all technologies that we’re evaluating. We’re particularly interested in Contributors that can act as a focal point for tapping relevant expertise from their organizations and colleagues in order to engage with CNCF discussions in a timely manner.
 
 The TOC already has the pattern of encouraging non-members to make non-binding votes, so no change in the TOC charter is necessary to allow Contributors.
 
```

Comment:
"so no change in the TOC charter is necessary to allow Contributors." is confusing. We could just change it to "the TOC encourages non-TOC-members to make non-binding votes".

Reply:
+1, there are a lot of lines below which continue use the word `Contributors`. We can probably just do a s/Contributors/non-TOC members like @cathyhongzhang suggested.

Reply:
@nikhita could you please go ahead and file a new PR to help clean this up further?

for this PR, i changed the language to `The TOC welcomes non-binding votes from members of the CNCF community to express their viewpoints.`

thanks!

Reply:
> @nikhita could you please go ahead and file a new PR to help clean this up further?

Will do it as a follow-up after this PR merges. :+1: